packages/utxo-global/src/btc/index.ts

```diff
@@ -0,0 +1,46 @@
+import { ccc } from "@ckb-ccc/core";
+import { Provider } from "../advancedBarrel";
+
+export class UtxoGlobalBTCSigner extends ccc.SignerBtc {
+  constructor(
+    client: ccc.Client,
+    public readonly provider: Provider,
+  ) {
+    super(client);
+  }
+
+  async getBtcAccount() {
+    const accounts = await this.provider.getAccount();
+    return accounts[0];
+  }
+
+  async getBtcPublicKey(): Promise<ccc.Hex> {
+    const pubKeys = await this.provider.getPublicKey();
+    const account = await this.getBtcAccount();
+    const pubKey = pubKeys.find((_pubKey) => _pubKey.address === account);
+    return ccc.hexFrom(pubKey?.publicKey!);
```

Comment:
It's better to throw an error in unexpected situations when not forced by type explicitly.

Reply:
fixed